Enhancements_Fast_api/main.py

```diff
@@ -0,0 +1,248 @@
+from fastapi import FastAPI, status, HTTPException, Depends
+from sqlalchemy.orm import Session
+from sqlalchemy import func, asc, desc
+import requests
+from database import SessionLocal, engine
+from fastapi import Query
+from typing import Optional, List
+
+import models
+from schemas import (
+    PokemonPostPutInputSchema,
+    PokemonPatchInputSchema,
+    PokemonGetOutputSchema,
+    PokemonPostPatchPutOutputSchema,
+)
+from typing import List
+
+app = FastAPI()
+
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+# Routes
+@app.get(
+    "/pokemon_all",
```

Comment:
this route is not reuired
we already have 
```python
@app.get( "/pokemon/", )
```

Reply:
I have removed this route.